### PR TITLE
Fix using source name to CHGOBJD for text description

### PR DESCRIPTION
**Pull request by edmundreinhardt**
use target object name instead of source name, allowing source name to be different

### CODE REVIEW

**Comment by edmundreinhardt** [comment]
fixes issue 340

File: src/mk/def_rules.mk [hunk 1]
```diff
@@ -1025,9 +1025,9 @@ endef
 define TABLE_TO_FILE_RECIPE = 
 	$(FILE_VARIABLES)
 	$(eval d = $($@_d))
-	@$(call echo_cmd,"=== Creating SQL TABLE from Sql statement [$(notdir $<)] in $(OBJLIB)")
+	@$(call echo_cmd,"=== Creating SQL TABLE $(OBJLIB)/$(basename $(notdir $@)) from Sql statement [$(notdir $<)]")
```
Comment (SanjulaGanepola) [inline src/mk/def_rules.mk:1028]: For consistency, shouldn't we update all the other recipes below to also echo this same information?

**Review by SanjulaGanepola — CHANGES_REQUESTED** [review]

File: src/mk/def_rules.mk (hunk 1 above)
Comment (edmundreinhardt) [reply src/mk/def_rules.mk:1028]: agreed

Comment (SanjulaGanepola) [reply src/mk/def_rules.mk:1028]: Addressed this in my latest commit

**Comment by SanjulaGanepola** [comment]
@edmundreinhardt Addressed the comment that I had. I assume this is safe to merge?

**Review by SanjulaGanepola — APPROVED** [review]

**Pull request merged**